src/main/resources/org/synyx/urlaubsverwaltung/core/mail/remind_currently_left_vacation_days.ftl

```diff
@@ -0,0 +1,3 @@
+Hallo ${recipientNiceName},
+
+Du hast noch ${vacationDaysLeft} Tage Urlaub für dieses Jahr offen.
```

Comment:
Paar Gedanken:
* Links in die UV anbieten
* Ist hier eine Statistik interessant?

Reply:
.. Tag(e) ...